spec/controllers/markdown_controller_spec.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+describe MarkdownController do
+  context 'POST' do
+    before :each do
+      @input = 'input'
+    end
+    let :submit do
+      get :explanation, params: { preview_input: @input }
+    end
+    context 'staff' do
+      before :each do
+        when_current_user_is :staff
+      end
+      it 'passes preview input parameter through as an instance variable' do
+        submit
+        expect(assigns[:preview_input]).to eql @input
```

Comment:
This is OK, I guess. Although I have doubts about how much we should be continuing to write assertions about instance variable assignments.

I kinda would prefer a system test that makes the request and asserts that the rendered text is on the page.

Reply:
I can possibly write a feature test for now? because the setup for the system test is in another branch.

Reply:
It does make sense to test the rendering on the site itself, that is a better test for sure.

Reply:
Yes, sorry.  Write an integration test of some sort. Which sort depends on which PR gets merged first, I guess.